job-server/src/main/resources/application.conf

```diff
@@ -207,7 +207,7 @@ spark {
       #Any locations of python libraries to be included on the PYTHONPATH when running Python jobs
       paths = []
       #The shell command to run when launching the subprocess for python jobs
-      executable = "python"
+      executable = "python3"
```

Comment:
Based on what your default python is on your system, executing `python` should run a specific version of python. I am trying to understand why we cannot keep python here and let user decide what the python version should be used.

Reply:
I have both python 2 and 3 installed on my machine and I simply want python 3 to be selected by default and not to mess with configuration files. My assumption was, that this holds true for most users nowadays.

Reply:
Yea, your assumption is correct.
This is a breaking change for existing jobserver instances. Maybe this should be added in release notes.

Reply:
Ok, I will add it